JiraReporter/AuthorsProcessing.cs

```diff
@@ -0,0 +1,383 @@
+﻿using JiraReporter.Model;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+using RestSharp;
+
+namespace JiraReporter
+{
+    class AuthorsProcessing
```

Comment:
Nu prea am de unde sa il sterg. Nu exista fisierul. Am dat mai demult un commit aici, probabil de asta apare. Commitul asta e vechi de vreo 6 zile. Am mai dat acum merge din master in branch. Fisierul asta nu il gasesc nicaieri.